superset-frontend/src/views/CRUD/utils.tsx

```diff
@@ -322,3 +322,7 @@ export const CardStyles = styled.div`
     text-decoration: none;
   }
 `;
+
+export const HOMEPAGE_CHART_FILTER = 'homepage_chart_filter';
```

Comment:
The utils file looks like a weird place to me to put these constants

Reply:
This is where we have been dumping most of the utils, const vars into for the crud views

Reply:
Can we add a comment describing what these constants are and how they're sued. Or perhaps move them to a file/module called `storage-keys` or something descriptive. I guess there's  already the `localStorageHelpers` module, perhaps it would make sense to store these constants there.

Reply:
There seems to be quite a few instances of `getFromLocalStorage`/`setInLocalStorage` throughout the codebase. Each one is using a different approach for managing keys (userId, enum, and now constants). It would be good to have a unified approach around this.

Reply:
I agree @nytai we should find a good unified approach that handles all cases.